Documentation/guides/messages/xa1024.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Xamarin.Android warning XA1024
+description: XA1024 warning code
+ms.date: 06/19/2020
+---
+# Xamarin.Android warning XA1024
+
+## Example messages
+
+```
+warning XA1024: The 'EnableProguard' MSBuild property is set to 'true' and the 'AndroidLinkTool' MSBuild property is empty, so 'AndroidLinkTool' will default to 'proguard'.
+error XA1011: Using ProGuard with the D8 DEX compiler is no longer supported. Please set the code shrinker to 'r8' in the Visual Studio project property pages or edit the project file in a text editor and set the 'AndroidLinkTool' MSBuild property to 'r8'.
```

Comment:
All in all, the new messages might be slightly excessive because they always appear alongside the XA1011 message, which does say to set `$(AndroidLinkTool)` to `r8`, and that will indeed resolve the error in call cases. So these new warnings are catching a fairly slim scenario where a user wants to try to disable code shrinking entirely instead of moving to `r8`. On top of that, since these new messages are warnings, if the user happens to have warnings temporarily hidden in the Error List, they wouldn't even see them alongside the XA1011 error.

I think there's a chance the new warnings could still help a few users here and there as DX moves closer to deprecation, but I'd also be OK with skipping this PR if it does indeed seem excessive. These messages will all be irrelevant anyway once DX compatibility is removed.